### PR TITLE
Update oxide to v2.0-beta

### PR DESCRIPTION
**Pull request by Eeems**

### CODE REVIEW

**Comment by Eeems** [comment]
Current error the build is getting is from the following:
> WARNING: Failure to find: ../../shared/dbussettings.h

It works fine when building on my local box though, and the file exists in the repository. Thoughts?

File: package/oxide/package [hunk 1]
```diff
@@ -2,22 +2,16 @@
 pkgname=oxide
 pkgdesc="Launcher application"
 url=https://github.com/Eeems/oxide
-pkgver=2.0~alpha-3
-timestamp=2020-09-11T05:33Z
+pkgver=2.0~beta~1
+timestamp=2020-09-26T20:53Z
 section=launchers
 maintainer="raisjn <of.rajsn@gmail.com>"
```
Comment (raisjn) [inline package/oxide/package]:
```suggestion
maintainer="Eeems <eeems@eeems.email>"
```

**Review by raisjn — COMMENTED** [review]
why did you delete tarnish.service? i thought we were going to start checking in service and draft files to the repo for some reason

**Comment by Eeems** [comment]
> why did you delete tarnish.service? i thought we were going to start checking in service and draft files to the repo for some reason

Because the source service file already has the conflicts in it.

**Review by raisjn — APPROVED** [review]

**Pull request merged**